### PR TITLE
update example notebook; bugfixes

### PR DESCRIPTION
**Pull request by nreinicke**
Supersedes #131 by merging that notebook with the notebook on the main branch.

Also includes a few fixes found along the way:
 - injects distance from the time model until we have a distance model in the scope for the energy traversal model
 - fixes the units for the speed coming out of the time model and into the energy model.

### CODE REVIEW

**Comment by robfitzgerald** [comment]
cool, thanks for superseding. did you look into whether grade issues were a problem here too, or is that just elsewhere?

**Comment by nreinicke** [comment]
I did double check grade in this case and everything looked okay, (we download the grade as decimal in this case)

File: python/nrel/routee/compass/io/generate_dataset.py [hunk 1]
```diff
@@ -73,26 +73,26 @@ def generate_compass_dataset(
     agg = agg if agg is not None else np.mean
 
     # pre-process the graph
-    log.info("processing graph topology and speeds")
+    print("processing graph topology and speeds")
```
Comment (robfitzgerald) [inline python/nrel/routee/compass/io/generate_dataset.py:76]: but what about logging? maybe we just need to be sure to set up logging.basicConfig correctly in the python library?

**Review by robfitzgerald — APPROVED** [review]
dig it. fun added explorations! i want to defend logging but don't want to get in the way of progress. 🥂

File: python/nrel/routee/compass/io/generate_dataset.py (hunk 1 above)
Comment (nreinicke) [reply python/nrel/routee/compass/io/generate_dataset.py:76]: yeah there's probably the _right_ way to do it that I didn't have the time to investigate; the motivation was that it was not logging to the jupyter notebook and so it was hard to tell what was happening.

**Pull request merged**